package.json

```diff
@@ -63,9 +61,9 @@
     "build-docs": "./build_helpers/buildAPIDocs.sh",
     "publish-site": "./build_helpers/publishStaticSite.sh",
     "publish-package": "./build_helpers/publishPackage.sh",
-    "test": "mocha-webpack --webpack-config webpack.config-test.js \"src/**/*-test.js\" --require build_helpers/test-globals.js",
-    "test:watch": "mocha-webpack --webpack-config webpack.config-test.js --watch \"src/**/*-test.js\" --require build_helpers/test-globals.js",
-    "test:server": "webpack-dev-server --config webpack.config-test.js --hot --inline"
```

Comment:
I do not know how important this feature is to run the tests in the browser. But as there is no such thing for jest I deleted this task. Actually I could also remove webpack.config-test.js